src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -271,6 +271,38 @@ public void onInstallShiny(InstallShinyEvent event)
                 new Command() { public void execute() {}});
    }
    
+   public void withDataImport(String userAction, final Command command)
```

Comment:
This is going to probably end up changing to withReadr, since each of the data import commands will have their own distinct dependencies.